File: app/src/main/java/com/github/kongpf8848/rxhttp/sample/bean/Feed.java

package com.github.kongpf8848.rxhttp.sample.bean;


import java.util.List;


public class Feed {

    private String date;
    private List<Story> stories ;
    private List<TopStory> top_stories ;


    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public List<Story> getStories() {
        return stories;
    }

    public void setStories(List<Story> stories) {
        this.stories = stories;
    }

    public List<TopStory> getTopStories() {
        return top_stories;
    }

    public void setTopStories(List<TopStory> topStories) {
        this.top_stories = topStories;
    }


}

File: app/src/main/java/io/github/kongpf8848/rxhttp/sample/bean/Feed.kt
package io.github.kongpf8848.rxhttp.sample.bean

import java.io.Serializable

data class Feed(
        var date: String,
        var stories: List<Story>? = null,
        var top_stories: List<TopStory>? = null
) : Serializable

data class Story(
        var id: Long = 0L,
        var type: Int = 0,
        var title: String? = null,
        var url: String? = null,
        var images: List<String>? = null,
        var hint: String? = null,
        var ga_prefix: String? = null,
        var image_hue: String? = null
) : Serializable

data class TopStory(
        var id: Long = 0L,
        var type: Int = 0,
        var title: String? = null,
        var url: String? = null,
        var image: String? = null,
        var hint: String? = null,
        var ga_prefix: String? = null,
        var image_hue: String? = null
) : Serializable